rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerRestClient.java

```diff
@@ -113,7 +113,12 @@ public JobIdData submitXml(String sessionId, InputStream jobXml) throws Exceptio
     }
 
     public JobIdData submitXml(String sessionId, InputStream jobXml, Map<String, String> variables) throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)